packages/cli/snapshots/integration/generators/relation.has-many.integration.snapshots.js

```diff
@@ -307,123 +307,7 @@ export class CustomerClassOrderClassController {
     @param.path.number('id') id: number,
     @param.query.object('where', getWhereSchemaFor(OrderClass)) where?: Where<OrderClass>,
   ): Promise<Count> {
-    return this.customerClassRepository.orderClasses(id).delete(where);
-  }
-}
-
-`;
-
-exports[
```

Comment:
Hmm any reason removing this exports?

Reply:
A snapshot matches one test case. This one is the test case that I removed.